src/TextFieldCatalog.js

```diff
@@ -99,8 +102,13 @@ const TextFieldCatalog = () => (
 );
 
 export const TextFieldHero = () => {
+  this.clickEvent = (label) => ReactGA.event({category: 'Component Interaction', action: 'Text Field Click', label: label});
```

Comment:
typically a functional component doesn't have member functions. Just define the function outside of this scope.